src/pages/DetailsPage.jsx

```diff
@@ -1,9 +1,19 @@
 import React from "react";
 import Heading from "../components/Heading";
+import Breadcrumps from "../components/Breadcrump";
 
 function DetailsPage() {
   return (
     <div>
+      <div className="breadcrumps-container">
+        <Breadcrumps variant="" page="details">
```

Comment:
empty string brings no additional value to the component, so it can be removed
```suggestion
        <Breadcrumps page="details">
```

Reply:
ok, done